src/redux/actions/common.js

```diff
@@ -0,0 +1,14 @@
+import { START_FETCH, SERVER_ERROR, CLEAR_ERROR } from "../action_types";
+
+export const startFetch = () => ({
+  type: START_FETCH
+});
+
+export const clearError = () => ({
+  type: CLEAR_ERROR
+});
+
+export const serverError = error => ({
+  type: SERVER_ERROR,
+  error
+});
```

Comment:
Add comments to the above action creators as well with a little more detail as they are being used by everybody.

Reply:
Added the comments. Is this what you meant?

Reply:
Now that's a message! Wrong place though